test/models/system_variable_test.exs

```diff
@@ -0,0 +1,118 @@
+defmodule CoursePlanner.SystemVariableTest do
```

Comment:
please break these tests apart, if you want to group them because they are kinda related you can use the `describe`